addons/website_slides/static/src/js/slides_course_page.js

```diff
@@ -24,11 +24,11 @@ export const SlideCoursePage = publicWidget.Widget.extend({
      * Collapse the next category when the current one has just been completed
      */
     collapseNextCategory: function (nextCategoryId) {
-        const $categorySection = document.getElementById(`category-collapse-${nextCategoryId}`);
-        const $categorySlides = document.querySelector(`ul[id=collapse-${nextCategoryId}]`);
-        if ($categorySection.getAttribute('aria-expanded') === 'false' && !$categorySlides.classList.contains('show')) {
-            $categorySection.setAttribute('aria-expanded', true);
-            $categorySlides.classList.add('show')
+        const categorySection = document.getElementById(`category-collapse-${nextCategoryId}`);
+        const categorySlides = document.querySelector(`ul[id=collapse-${nextCategoryId}]`);
+        if (categorySection?.getAttribute('aria-expanded') === 'false' && !categorySlides.classList.contains('show')) {
+            categorySection.setAttribute('aria-expanded', true);
+            categorySlides.classList.add('show')
```

Comment:
The problem with this is are
* it is not explicit that categorySlides will exist if categorySection exists
* We query for categorySlides even when categorySection already doesn't exist

As you confirmed that cSlides cannot exist without cSection, and as we can safely "add" a class even if it is already there, I'd suggest the shorter and more straightforward following

```suggestion
        if (categorySection?.getAttribute('aria-expanded') === 'false' ) {
            categorySection.setAttribute('aria-expanded', true);
            document.querySelector(`ul[id=collapse-${nextCategoryId}]`).classList.add('show');
```